OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -231,7 +232,17 @@ public void Tick(World world)
 				foreach (var a in actors)
 				{
 					var adjustedModifiers = args.DamageModifiers.Append(GetFalloff((args.Source - a.CenterPosition).Length));
```

Comment:
Unrelated to this PR, but... am I reading this wrong, or does damage fall off the farther the hit actor is from the source actor, instead of the farther it is from the beam's center (which would be the correct behavior)?

Reply:
It looks like the desired behaviour according to #9357 was to have a certain section of the beam between `headPos` and `x` that deals maximum damage, then a linear falloff in damage between `x` and `y`, and then the remaining section from `y` to `tailPos` deal minimum damage.